src/electron/tools/file/registry.ts

```
@@ -0,0 +1,122 @@
+  deleteFileOrDirectoryFD,
+  getFileInfo,
+  getFileInfoFD,
+  checkExists,
+  checkExistsFD,
+  searchFiles,
+  searchFilesFD,
+} from "./tools.js";
+
+export interface ToolRegistry {
+  [key: string]: {
+    function: (params: any) => Promise<string>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)